packages/ember-htmlbars/tests/helpers/each_test.js

```diff
@@ -1,617 +0,0 @@
-import { context } from 'ember-environment';
-import EmberObject from 'ember-runtime/system/object';
-import run from 'ember-metal/run_loop';
-import EmberView from 'ember-views/views/view';
-import { A as emberA } from 'ember-runtime/system/native_array';
-import EmberController from 'ember-runtime/controllers/controller';
-
-import { set } from 'ember-metal/property_set';
-import { runAppend, runDestroy } from 'ember-runtime/tests/utils';
-
-import compile from 'ember-template-compiler/system/compile';
-
-import { registerKeyword, resetKeyword } from 'ember-htmlbars/tests/utils';
-import viewKeyword from 'ember-htmlbars/keywords/view';
-
-import buildOwner from 'container/tests/test-helpers/build-owner';
-import { OWNER } from 'container/owner';
-
-var people, view, owner;
-var template, templateMyView, MyView, MyEmptyView, templateMyEmptyView;
-var originalViewKeyword;
-
-var originalLookup = context.lookup;
-var lookup;
-
-import isEnabled from 'ember-metal/features';
-if (!isEnabled('ember-glimmer')) {
-  // jscs:disable
-
-QUnit.module('the #each helper', {
-  setup() {
-    context.lookup = lookup = {};
-
-    originalViewKeyword = registerKeyword('view',  viewKeyword);
-
-    template = compile('{{#each view.people as |person|}}{{person.name}}{{/each}}');
-    people = emberA([{ name: 'Steve Holt' }, { name: 'Annabelle' }]);
-
-    owner = buildOwner();
-
-    owner.register('view:toplevel', EmberView.extend());
-
-    view = EmberView.create({
-      [OWNER]: owner,
-      template: template,
-      people: people
-    });
-
-    templateMyView = compile('{{name}}');
-    lookup.MyView  = MyView = EmberView.extend({ template: templateMyView });
-    owner.register('view:my-view', MyView);
-
-    templateMyEmptyView = compile('I\'m empty');
-    lookup.MyEmptyView = MyEmptyView = EmberView.extend({
-      template: templateMyEmptyView
-    });
-    owner.register('view:my-empty-view', MyEmptyView);
-
-    runAppend(view);
-  },
-
-  teardown() {
-    runDestroy(owner);
-    runDestroy(view);
-    owner = view = null;
-
-    context.lookup = originalLookup;
-
-    resetKeyword('view', originalViewKeyword);
-  }
-});
-
-var assertHTML = function(view, expectedHTML) {
-  var html = view.$().html();
-
-  // IE 8 (and prior?) adds the \r\n
-  html = html.replace(/<script[^>]*><\/script>/ig, '').replace(/[\r\n]/g, '');
-
-  equal(html, expectedHTML);
-};
-
-QUnit.test('it renders the template for each item in an array', function() {
-  assertHTML(view, 'Steve HoltAnnabelle');
-});
-
-QUnit.test('it updates the view if an item is added', function() {
-  run(function() {
-    people.pushObject({ name: 'Tom Dale' });
-  });
-
-  assertHTML(view, 'Steve HoltAnnabelleTom Dale');
-});
-
-QUnit.test('it updates the view if an item is removed', function() {
-  run(function() {
-    people.removeAt(0);
-  });
-
-  assertHTML(view, 'Annabelle');
-});
-
-QUnit.test('it updates the view if an item is replaced', function() {
```

Comment:
Added to by [{{#each as}} module](https://github.com/emberjs/ember.js/pull/13349/commits/c08e75f3474e8bb995f5f24b837479bf3fca64f8#diff-d3c97c178f7e01e59143b5bf2f84062aR93)'s INUR